nixos/modules/services/misc/zoneminder.nix

```diff
@@ -49,7 +49,7 @@ let
     # Database
     ZM_DB_TYPE=mysql
     ZM_DB_HOST=${cfg.database.host}
-    ZM_DB_NAME=${cfg.database.name}
+    ZM_DB_NAME=${if cfg.database.createLocally user else cfg.database.name}
```

Comment:
There is syntax problem here - `then` is missing. I guess this particular change wasn't tested somehow.

Reply:
Yup, sorry about that. Will fix that asap, thanks for looking into it.